extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/supports/context/GrpcDuplicatedContextGrpcInterceptor.java

```diff
@@ -84,7 +84,7 @@ private synchronized ServerCall.Listener<ReqT> getDelegate() {
                     delegate = supplier.get();
                 } catch (Throwable t) {
                     // If the interceptor supplier throws an exception, catch it, and close the call.
-                    log.warn("Unable to retrieve gRPC Server call listener", t);
+                    log.warn("Unable to retrieve gRPC Server call listener");
```

Comment:
Why do you drop the cause? With warnf and t as first parameter it would print the stack trace of the cause.

Reply:
The cause is already logged at `close` ... so didn't wan't to log it twice.